### PR TITLE
Revoke Trust

### PR DESCRIPTION
**Pull request by pogopaule**
see https://github.com/yunity/karrot-frontend/pull/2352

### CODE REVIEW

File: karrot/groups/api.py [hunk 1]
```diff
@@ -201,6 +201,30 @@ def trust_user(self, request, pk, user_id):
 
         return Response(data={})
 
+    @action(
+        detail=True,
+        methods=['POST'],
```
Comment (nicksellen) [inline karrot/groups/api.py:206]: As mentioned in the frontend PR, using `DELETE` method on the same endpoint as the trust one would fit in better with what we've done so far.

**Review by nicksellen — COMMENTED** [review]
Great, looks really good to me :100: 

... just the URL endpoint question open.

File: karrot/groups/api.py (hunk 1 above)
Comment (pogopaule) [reply karrot/groups/api.py]: done

File: karrot/groups/api.py [hunk 2]
```diff
@@ -201,6 +201,23 @@ def trust_user(self, request, pk, user_id):
 
         return Response(data={})
 
+    @trust_user.mapping.delete
```
Comment (nicksellen) [inline karrot/groups/api.py:204]: Ah, nice find using this [`mapping` thing](https://www.django-rest-framework.org/api-guide/viewsets/#routing-additional-http-methods-for-extra-actions), we could clean up other actions using that pattern one day! :+1:

**Review by nicksellen — APPROVED** [review]
Looks good to me now! :rocket:

**Comment by pogopaule** [comment]
@nicksellen running `python manage.py makemessages` on my machine does not cause `karrot/locale/en/LC_MESSAGES/django.po` to change. But on circleci it does and fails the test pipeline...do you have an idea why?

**Comment by nicksellen** [comment]
Have you run `./sync.py` to install latest dependencies? Even patch version differences can generate slightly different locale files, and I just merged an dependency update that was impacted in that way. I'm assuming you also just rebased on master.

**Comment by pogopaule** [comment]
@nicksellen Thank you! `./sync.py` did the job!

File: karrot/history/models.py [hunk 3]
```diff
@@ -43,6 +43,7 @@ class HistoryTypus(enum.Enum):
     ACTIVITY_TYPE_CREATE = 25
     ACTIVITY_TYPE_MODIFY = 26
     ACTIVITY_TYPE_DELETE = 27
+    USER_LOST_EDITOR_ROLE = 18
```
Comment (nicksellen) [inline karrot/history/models.py:46]: I just noticed this number is not unique, I guess was supposed to be `28` :)

Comment (pogopaule) [reply karrot/history/models.py:46]: Oh no 🤦 I'll fix that tomorrow in a new PR

**Pull request merged**